config/global.php

```diff
@@ -72,4 +72,16 @@
                 'tld' => false,
             ))));
     },
+
+    'tracker.request.processors' => array(),
```

Comment:
We could add a comment saying it holds an array of RequestProcessor instances.

> Also can we avoid that a plugin developer has to add it here? How can we make it super simple for a plugin developer?

Why would it be difficult for a plugin developer to add items to DI? DI is not an implementation detail of Piwik it is the new base of Piwik's architecture. Plugin developers should be aware of it and how to use it.